packages/data/lib/services/app_interaction_service_impl.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:domain/services/app_interaction_service.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class AppInteractionServiceImpl implements AppInteractionService {
+  static const _lastAppInteractionTimeKey = 'last_app_iteraction_time';
```

Comment:
let's keep interactions also in db, to have more consistent flow

Reply:
Fixed.